state-sync/src/request_manager.rs

```diff
@@ -326,6 +324,22 @@ impl RequestManager {
         }
     }
 
+    fn get_network_sender(&mut self, peer: &PeerNetworkId) -> StateSyncSender {
+        self.network_senders
+            .get_mut(&peer.network_id())
+            .unwrap_or_else(|| panic!("Missing network sender for peer: {:?}", peer))
```

Comment:
As in my previous comment, this is kinda a weird message that doesn't match what a network sender is.

Perhaps better is `panic!("Missing network sender for network:  {:?}", peer.network_id())`.  The original author didn't really take into account that the error message should match the value that it's missing on `NetworkId` (In this case `NodeNetworkId`, but I'm trying to get rid of that one)